README.md

```diff
@@ -6,12 +6,14 @@ The release notes of `carbon-react` will indicate which codemod you should use.
 ## Usage
 
 ```
-npx carbon-codemod <name-of-codemod> <target>
+npx carbon-codemod <name-of-codemod> <target> <additional-arguments-if-required-by-codemod>
 ```
 
 - [`button-destructive`](./transforms/button-destructive)
 - [`deprecate-create`](./transforms/deprecate-create)
 - [`message-remove-classic-theme`](./transforms/message-remove-classic-theme)
+- [`rename-prop (requires additional <component> <old> <replacement> arguments)`](./transforms/rename-prop)
```

Comment:
I don't think this `(requires additional ... )` bit is needed as these should link through to the readme for the individual codemods, which should explain that in their "Usage" section

Reply:
Yeah lets simplify this and revert the changes to line 9 too.